### ISSUE TITLE
maven repo url changed

### ISSUE THREAD

**Issue by bnwlf** [TheRoddyWMS/Roddy#348]
Hi,
the maven repo url which is set in line 88 to 92 in the _build.gradle_ file isn't up to date.
 

```
repositories {

    mavenCentral()                                        // Most dependencies

    maven { url 'https://jitpack.io' }                    // Github: RoddyToolLib, BatchEuphoria

    maven { url "http://central.maven.org/maven2/" }      // Some of the remaining dependencies

}
```

The servers are now days only reachable via https and a new url.
`   maven { url "https://repo1.maven.org/maven2/" }      // Some of the remaining dependencies`
This change fixes the problem.
**Cheers!**

**Comment by vinjana** [comment]
Cool! Thanks for you contribution. I will directly change this!

**Comment by vinjana** [comment]
Fixed with tag a803975
